tmux_window_name.tmux

```diff
@@ -9,4 +9,8 @@ if ! echo "$pip_list" | grep libtmux -q; then
 fi
 
 tmux set -g automatic-rename off
-tmux set-hook -g 'after-select-window[8921]' "run-shell "$CURRENT_DIR/scripts/rename_session_windows.py""
+tmux set-hook -g 'after-select-window[8921]' "run-shell ""$CURRENT_DIR""/scripts/rename_session_windows.py"
+tmux set-hook -g 'after-rename-window[8921]' 'set -uw @tmux_window_name_enabled'
+tmux set-hook -g 'after-new-window[8921]' 'set -w @tmux_window_name_enabled 1'
+
+tmux bind '"' 'set -w @tmux_window_name_enabled 1; run-shell '"$CURRENT_DIR"'/scripts/rename_session_windows.py'
```

Comment:
I don't like that a bind is needed for re-enabling the plugin, maybe we can detect when a user renames the window to an empty name and that will re-enable the plugin?